solr/CHANGES.txt

```diff
@@ -51,6 +51,8 @@ New Features
 
 * SOLR-16596: Learning To Rank - Added support for null feature values in multiple additive trees models (Anna Ruggero via Alessandro Benedetti)
 
+* SOLR-15955: Upgrade to Jetty 10.x (Mark Miller, Kevin Risden)
+
```

Comment:
Not sure that under `"New Feature"` makes sense, but figured it was good to call out up top.

Reply:
Definitely not "New Feature" for bumping up a dependency unless somehow the user gets some new ability here.

Reply:
Thats fair - the only existing place that made rough sense was "Other Changes" or "Build" based on the existing categories being:

* New Features
* Improvements
* Optimizations
* Bug Fixes
* Build
* Other Changes

Reply:
Typically goes in Other Changes.  Could be justified in some other categories if we expect an appreciable difference.
Obviously this is a big dependency upgrade so it should be called out when we do the release notes, maybe mentioning some benefits if you know of any in particular that matter.

Reply:
moved to other changes